src/main/java/uk/gov/justice/probation/courtcaseservice/service/OffenderMatchService.java

```diff
@@ -87,6 +87,20 @@ public OffenderMatchDetailResponse getOffenderMatchDetailsByCaseIdAndDefendantId
         return OffenderMatchDetailResponse.builder().offenderMatchDetails(offenderMatchDetails).build();
     }
 
+    public OffenderMatchDetailResponse getOffenderMatchDetailsByDefendantId(String defendantId) {
```

Comment:
This is returning a DTO from the service layer, I appreciate the previous method does the same but in general it's probably preferable to return Entities and have the controller do the mapping so these methods are reusable.

Reply:
Done, thanks